packages/core/src/collections/notes.ts

```diff
@@ -176,6 +176,15 @@ export class Notes implements ICollection {
     return note;
   }
 
+  async tags(id: string) {
+    return this.db.relations
+      .to({ id, type: "note" }, "tag")
+      .selector.items(undefined, {
+        sortBy: "dateCreated",
+        sortDirection: "asc"
+      });
+  }
+
```

Comment:
I've moved the `getTags` function here, do let me know if this is okay or should I revert it

Reply:
Good idea.